docs/Cloud Integration.md

```diff
@@ -61,15 +61,67 @@ Azure supports a multiple mechanism for authentication through environment varia
 
 # Usage
 
-Before running helm, the environment variable `HELM_SECRETS_BACKEND=vals` needs to be set.
-This enables the vals integration in helm secrets.
+Before running `helm`, the environment variable `HELM_SECRETS_BACKEND=vals` needs
+to be set or the command line option `--backend=vals` must be put in use.
+
+This enables the `vals` integration in helm-secrets.
+
 Vals needs cloud prover credentials to fetch secrets from the secret services.
+Be sure to have them in place before trying to use (for instance, use the cloud
+provider own CLI to fetch the same secrets).
 
 helm-secrets can not fill the cloud provider secrets store through the encryption command.
 
+> :warning: Vals reference strings must be declared in the "values" file (the
+> YAML file being used by the Helm template to provide values), not in the
+> resource itself!
```

Comment:
Note: This is partially true. Between your original issue and the PR, I introduce a new feature which make it possible to render vals reference from templates. https://github.com/jkroepke/helm-secrets/wiki/Usage#evaluate-secret-reference-inside-helm-template